buildSrc/build.gradle

```diff
@@ -55,6 +55,7 @@ dependencies {
   runtime "org.ajoberstar.grgit:grgit-gradle:3.0.0"                                                   // Enable website git publish to asf-site branch
   runtime "com.avast.gradle:gradle-docker-compose-plugin:0.8.8"                                       // Enable docker compose tasks
   runtime "ca.cutterslade.gradle:gradle-dependency-analyze:1.3.0"                                     // Enable dep analysis
+  runtime "gradle.plugin.net.ossindex:ossindex-gradle-plugin:0.4.8"
```

Comment:
Any reason not to use latest (0.4.11)?

Reply:
reason being I did this branch a while ago :) thanks for pointing out.

Reply:
1 updated the PR